modules/upgrade/pages/server-maintenance-uyuni.adoc

```diff
@@ -0,0 +1,47 @@
+[server-maintenance]
+= Server - Maintenance =
+
+Between regular {productname} releases packages the openSUSE Leap team releases package updates to improve the system and to fix security issues.
+It is recommended to install these package update in due time.
```

Comment:
To be honest, I think you are talking here about OS Maintenance, am I right?

With the other section about "minor" section this is somehow confusing. Maybe you can somehow merge both adocs.

Remember that, in the end, we are a rolling release product, as explained below. That can be relevant.

Reply:
> To be honest, I think you are talking here about OS Maintenance, am I right?

mostly, but I think sometimes you also publish security fixes for Uyuni between the (rolling) releases?  Maybe not, we don't seem to have an update repo for stable.

But if you think it's confusing I can change it.

Reply:
merged into minor

Reply:
No, we don't maintenance of any kind between versions. It's said everywhere: It's pure rolling release and if a security fix or any kind of fix is needed, we release a new version.